.chloggen/servicesettings-confmap.yaml

```diff
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Technically adding a field to a struct is not a breaking change. See https://github.com/open-telemetry/opentelemetry-collector/blob/main/VERSIONING.md, we follow the go compatibility promise. We don't need it to be required, right?

Reply:
That's a good point, I think I may have forgot to change this when updating the `service.New` call. Updated.